corehq/apps/case_search/utils.py

```diff
@@ -54,20 +54,11 @@ def _get_initial_search_es(self):
         return search_es
 
     def _assemble_optional_search_params(self):
-        self._add_include_closed()
         self._add_xpath_query()
         self._add_owner_id()
         self._add_blacklisted_owner_ids()
         self._add_case_property_queries()
 
-    def _add_include_closed(self):
-        try:
-            include_closed = self.criteria.pop('include_closed')
```

Comment:
do you think this will cause an issue in apps that haven't been rebuilt after this change? Or is this property in `UNSEARCHABLE_KEYS` ?

Reply:
I'm reasonably sure no real apps have this flag turned on, even in old builds, although there are test apps that do. If anyone works with a test app, it'll still break if `include_closed` is in `UNSEARCHABLE_KEYS`, just more subtly (not returning closed cases instead of not returning anything).

Reply:
If I understand this correctly, https://github.com/dimagi/commcare-hq/blob/ccc18f4e452b4fbe3ec32046f12eec58154454c7/corehq/apps/app_manager/suite_xml/sections/remote_requests.py#L143-L146 every app with case claim will have this search key in the suite (set to False). 
It doesn't look like this key is in `UNSEARCHABLE_KEYS`, so won't this be included as a search parameter for every search request?

Reply:
ah, I get it now! Thank you, I'll update `UNSEARCHABLE_KEYS`.